config/global.js

```diff
@@ -41,6 +41,7 @@ const {
 	SC_DEMO_USER_NAME = 'schueler@schul-cloud.org',
 	SC_SUPERHERO_USER_PASSWORD = 'Schulcloud1!',
 	SC_SUPERHERO_USER_NAME = 'superhero@schul-cloud.org',
+	FEATURE_STUDENTS_CANT_BE_TEAM_ADMINISTRATOR,
```

Comment:
Please follow the name shema of feature ENVs (take a look at default.schema.js). Should also set to true

Reply:
add it also default.schema.json